src/cluster.h

```diff
@@ -127,6 +127,7 @@ typedef struct clusterNode {
                                     may be NULL even if the node is a slave
                                     if we don't have the master node in our
                                     tables. */
+    unsigned long long last_in_ping_message; /* The number of the last carried in the ping message */
```

Comment:
```suggestion
    unsigned long long last_in_ping_gossip; /* The number of the last carried in the ping gossip section */
```
Probably worth clarifying this in the gossip section, not just the message.

Reply:
thank you, done